docs/notary-signer-config.md

```diff
@@ -0,0 +1,224 @@
+<!--[metadata]>
++++
+title = "Notary Signer Configuration File"
+description = "Specifies the configuration file for Notary Signer"
+keywords = ["docker, notary, notary-signer, configuration"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Notary Signer Configuration File
+
+An example (full) server configuration file.
+
+```json
+{
+	"server": {
+		"http_addr": ":4444",
+		"grpc_addr": ":7899",
+		"cert_file": "./fixtures/notary-signer.crt",
```

Comment:
#304 Makes this `tls_cert_file` and `tls_key_file`, to be more consistent with notary-server.